includes/data-stores/class-wc-data-store-wp.php

```diff
@@ -630,4 +630,16 @@ public function delete_from_lookup_table( $id, $table ) {
 		);
 		wp_cache_delete( 'lookup_table', 'object_' . $id );
 	}
+
+	/**
+	 * Converts a WP post date string into a timestamp.
+	 *
+	 * @since 4.7.0
```

Comment:
This change won't make it to 4.7 so:

```suggestion
	 * @since 4.8.0
```